### PR TITLE
Updated the page /16-04/azure (copy doc)

### LINKED ISSUE #10416: Copy update on /16-04/azure
Please update `/16-04/azure` based on the [copy doc](https://docs.google.com/document/d/1V99KwimN8Hla8L_CikNc6I0amqj99cSrBsVlAjIkBqE/edit#)

### PR DESCRIPTION
**Pull request by goulinkh**
## Done

- Updated the content of https://ubuntu.com/16-04/azure

## QA

- Go to the webiste https://ubuntu-com-10420.demos.haus/16-04/azure
- Make sure that the changes in the [copy doc](https://docs.google.com/document/d/1V99KwimN8Hla8L_CikNc6I0amqj99cSrBsVlAjIkBqE/edit#) are applied

## Issue / Card

Fixes #10416

### CODE REVIEW

**Comment by bethcollins92** [comment]
Hey @goulinkh, there has been a couple more updates to the copy doc made today. Can you please make the changes then let @minkyngkm know when it's ready for review. It's high priority, needs to be merged today

File: templates/16-04/azure.html [hunk 1]
```diff
@@ -134,7 +135,7 @@ <h3>
         Microsoft and Canonical joint webinar on 16.04 migration strategies and options
       </h3>
       <p>
-        Microsoft and Canonical partnered on a joint webinar to help you through this evaluation process. The webinar covers the following topics:
+        Microsoft and Canonical partnered on a <a href="https://info.microsoft.com/ww-ondemand-expert-tips-to-easily-migrate-your-ubuntu-stack-to-azure.html">joint webinar</a> to help you through this evaluation process. The webinar covers the following topics:
```
Comment (minkyngkm) [inline templates/16-04/azure.html:138]: It seems we normally add `class="p-link--external"` when it goes to an external website
```suggestion
        Microsoft and Canonical partnered on a <a class="p-link--external" href="https://info.microsoft.com/ww-ondemand-expert-tips-to-easily-migrate-your-ubuntu-stack-to-azure.html">joint webinar</a> to help you through this evaluation process. The webinar covers the following topics:
```

**Review by minkyngkm — APPROVED** [review]

**Comment by minkyngkm** [comment]
Please add the class for the external link! Other than that, LGTM! @goulinkh 👍

**Comment by bethcollins92** [comment]
FYI @nmav

**Pull request merged**